### ISSUE TITLE
Energy Dashboard slow to response or freezing

### ISSUE THREAD

**Issue by samuelolteanu** [home-assistant/frontend#17889]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

I previously thought it was core related so i first opened this[ issue](https://github.com/home-assistant/core/issues/100031) in the Home Assistant Core section. However i've been advised this is looking like a frontend related thing so there I am.

My problem is I cannot navigate Energy Dashboard after more than fiew hours of any gathered data. This happens on mobile (browser or app) and to a less degree on desktop. If i do register back all energy entities relevant to me, after about a day, desktop performance will start to not be usable as well.  Nothing obvious in the logs. Not sure when the issue started but it's less than a month.

Things I tried so i might have isolated the problem:

- Purge all entities.
- New database, works fine at very first, when no data is showing
- Remove all energy entities, rename the id of essential ones and add them back on a clean database.
- Excluded non essential entities from recording
- Changed energy entities to send new data (delta) at/of 1kwh.
- Restored backups (sadly no generational, 1 month oldest). Frontend version still the latest version after any restore.
- Different core versions oldest being 2023.3.x
- Not do anything to it for a few days. Any change that i did make, I have waited for new data to show up.
- Changed ha host (different computer, different ssd, the old ssd with higher performance, less capacity , hyper-v (current) or ha as main os. Plenty of ram and storage, quad core CPUs.

- Stopped/deleted add-ons
- Change system language to English
-  **Deleted all cards, custom or not**
- Cleared caches, reinstalled app, lite or full, checked all relevant mobile browsers. 
- Restarted/factory reset mobile device Samsung A53.
- Other phones. Faster soc -> slightly more usable.

Might or might not be related: While general navigation, including graphs load up quickly, complex automation dialogs/action "folders" take longer than usual to expand. They were created on the same device and I noticed no such issues then.

This post is mainly intended to see if there are maybe other users who have a similar issue lately and obviously because I'm out of ideas. I've installed and configured [energy-flow-card-plus](https://github.com/flixlix/energy-flow-card-plus) so I've got some functionality back (it loads data like expected). I'm not starting a new HA install just yet (like i intended when I first posted).

https://github.com/home-assistant/frontend/assets/85267083/1c7f2214-2e54-48c3-830c-38bc31ee29ba

### Describe the behavior you expected

Energy dashboard usable on a mainstream device

### Steps to reproduce the issue

I simply don't know what to write here right now.

### What version of Home Assistant Core has the issue?

 2023.9.1

### In which browser are you experiencing the issue with?

Firefox 117.0, Chrome 116.0.5845 android or desktop, Edge, Companion 2023.8.2-full/lite

### Which operating system are you using to run this browser?

Windows 10/Android 13

**Comment by kongjudas** [comment]
Same problem here, causing my Android wall screen to run out of battery and shut down. Same effect in Safari on iMac and MacBook Pro. Machine heats up and browser "dies".

**Comment by kongjudas** [comment]
My frontend freeze on loading Lovelace view, before it's finished. This started after the 2023.9.2 update. There are no log to show for it.

**Comment by samuelolteanu** [comment]
Must write an update: it seems to be.....fine as of late. I simply ignored the issue and focused on something else like quantifying how much of a clear sky it is (multiliniar regression with sun attributes and a sunlight sensor...don't ask)....so i needed multiple days of data that meant changing recorder purge keep days from 1 to 7 and i'm hightly suspicios that's what fixed it. 
I checked after updating to core 2023.9.2(with its new frontend version)  and supervisor with the same name separately and nothing changed then (at least immediately).
Will change it back to 1 day once I finish my thing and report.
